project3/style.css

```diff
@@ -0,0 +1,209 @@
+:root {
+    /*  ==================
+      COLOR PALETTE
+      ==================*/
+
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+    /* Reminder that you can name your variables anything you want, they just have to start with two dashes (--) */
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+.button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+​ .button-reset[disabled] {
+    cursor: default;
+}
+
+fieldset {
+    border: 0;
+    padding: 0;
+}
+
+label {
+    display: block;
+}
+
+input,
+select,
+textarea {
+    font-family: inherit;
+    font-size: inherit;
+}
+
+select {
+    cursor: pointer;
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 3px solid var(--color-offwhite);
+    height: 12px;
+    width: 12px;
+    cursor: pointer;
+
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: aqua;
+    cursor: pointer;
+}
+
+input[type="radio"]:checked+label,
+input[type="checkbox"]:checked+label {
+    color: aqua;
+}
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="number"]:focus {
+    background-color: aqua;
+
+}
+
+input[type="text"]:required:valid,
+input[type="email"]:required:valid,
+input[type="number"]:required:valid {
+    background-color: greenyellow;
+
+}
+
+input[type="text"]:not(:placeholder-shown):invalid,
+input[type="email"]:not(:placeholder-shown):invalid,
+input[type="number"]:invalid {
+    background-color: var(--color-offwhite);
+
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+}
+
+body {
+    color: var(--color-offwhite);
+    /*margin-left: 100px;*/
+    font-family: 'Teko', sans-serif;
+    font-size: 20px;
+}
```

Comment:
This should be one of the first rulesets in your stylesheet -- here, I would expect it to be around the `button-reset` class. You want your `body` styles defined early because they provide a default for the rest of the document.